data/korath outfits.txt

```diff
@@ -148,6 +148,18 @@ outfit "Command Center"
 	"automaton" -1
 	description "In response to occasional raids by the robotic war machines left behind after their recent conflict, the Kor Efret have developed this minimal, spartan command center that can be installed in a captured robotic ship to allow a pilot to override the AI and interface with the ship's controls. The command center only provides a single bunk for the captain, so more may be necessary if the ship has turrets or other systems that require crew."
 
+
```

Comment:
For same-category outfits, the spacing appears to be 1 line (you've used 2 both before and after).

Reply:
I followed outfits.txt example where systems were sub-divided further. It probably isn't necessary here.

Reply:
Looks like your thought was correct, and there just wasn't a line between the Command Center and the system cores that there should be.

So perhaps add that space and move this outfit above the Command Center?
e.g.
core
>
>
this
>
>
command
>
control ...

Reply:
Done